docs/extensions-plugins/obsidian.mdx

```diff
@@ -192,15 +191,15 @@ Pieces adds **embedded buttons** into your markdown files to help you save, copy
 ---
 
 ## Pieces Copilot powered by qGPT - Personal Knowledge Assistant
-One of the most significant features of this plugin is Pieces Copilot, an artificial intelligence assistant designed to assist with your personal knowledge management.
+One of the most significant features of this plugin is the Pieces Copilot, an artificial intelligence assistant designed to assist with your personal knowledge management.
 The Copilot learns from the notes in your vault and adapts to the context of what you're working on. To try out this impressive technology: 
 
-- Ensure Pieces OS is open and updated to at least 5.1.2.
-- To view the Copilot, open the Pieces plugin and click the bot logo on the top slider to switch to Copilot.
-- The Copilot will use your entire vault as context for the chatbot. This means you can query your Obsidian vault with plain text and we will semantically match your question to any of your relevant notes.
+- Ensure Pieces OS is open and updated.
+- To view the Pieces Copilot, open the Pieces for Developers plugin and click the bot logo on the top slider to switch to Copilot.
+- The Pieces Copilot will use your entire vault as context for the chatbot. This means you can query your Obsidian vault with plain text and we will semantically match your question to any of your relevant notes.
```

Comment:
Do you want to also say that this is customizable?

Reply:
- The Pieces Copilot is also highly customizable, as a user you can select which notes you would like the Pieces Copilot to use as context, as well as selecting which LLM runtime you would like to use, local or cloud.